### ISSUE TITLE
[事前学習] - Reasoning ablations

### ISSUE THREAD

**Issue by stefanik12** [llm-jp/pretrain-tasks#296]
# Overview

Implement and execute continued pre-training with addressed architectural ablations
# Details

The goal of the project is to interpret the impact of recent architectural refinements (gated attention, alternative normalisations, different activations) on reasoning capabilities of resulting models in controlled settings to inform the creation of future architectures.

I will implement these as refinements in existing pre-trained architectures and assess the impact in continued pre-training (to minimise the necessary compute budget).

# Resources

* **計算機**
  * クラスタ: mdx llm-jp-nvlink
  * ノード種別: A100
  * ノード台数: 1-2
* **コード**
  * リポジトリ: https://github.com/stefanik12/reasoning_ablations
* **開始日**: 2025-01-01
* **終了予定日**: 2025-03-31 （バッファ期間を含む）
